raillabel_providerkit/validation/validate_dimensions/_dimensions.py

```diff
@@ -0,0 +1,218 @@
+
+            if annotation.attributes[attr_name] not in attr_val:
+                return False
+
+        return True
+
+    def _preprocess_variables(self) -> None:
+        if isinstance(self.type, str):
+            self.type = [self.type]
+
+        if self.attributes is None:
+            return
+
+        for attribute in self.attributes:
+            if isinstance(self.attributes[attribute], str):
+                self.attributes[attribute] = [self.attributes[attribute]]
+
+
+DIMENSIONS = [
+    _TypeDimensions(
+        type="person",
+        attributes={
+            "age": "adult",
+            "pose": "upright",
+        },
+        height=_DimensionRange(1.1, 2.7),
+        width=_DimensionRange(0.1, 2.0),
+    ),
+    _TypeDimensions(
+        type="person",
+        attributes={
+            "age": "adult",
+            "pose": "sitting",
+        },
+        height=_DimensionRange(0.5, 2.5),
+        width=_DimensionRange(0.1, 2.0),
+    ),
+    _TypeDimensions(
+        type="person",
+        attributes={
+            "age": "adult",
+            "pose": "lying",
+        },
+        height=_DimensionRange(0.1, 0.6),
+        width=_DimensionRange(0.1, 2.5),
+    ),
+    _TypeDimensions(
+        type="person",
+        attributes={
+            "age": "child",
+            "pose": "upright",
+        },
+        height=_DimensionRange(0.5, 1.6),
+        width=_DimensionRange(0.1, 1.5),
+    ),
+    _TypeDimensions(
+        type="person",
+        attributes={
+            "age": "child",
+            "pose": "sitting",
+        },
+        height=_DimensionRange(0.5, 1.4),
+        width=_DimensionRange(0.1, 1.5),
+    ),
+    _TypeDimensions(
+        type="person",
+        attributes={
+            "age": "adult",
+            "pose": "lying",
+        },
+        height=_DimensionRange(0.1, 0.6),
+        width=_DimensionRange(0.1, 1.6),
+    ),
+    _TypeDimensions(  # catchall for person
+        type="person",
+        height=_DimensionRange(0.1, 2.7),
+        width=_DimensionRange(0.1, 2.0),
+    ),
+    _TypeDimensions(
+        type="pram",
+        height=_DimensionRange(0.3, 1.6),
+        width=_DimensionRange(0.4, 2.0),
+    ),
+    _TypeDimensions(
+        type="crowd",
+        height=_DimensionRange(0.5, 2.5),
+    ),
+    _TypeDimensions(
+        type="scooter",
+        height=_DimensionRange(0.3, 1.6),
+        width=_DimensionRange(0.1, 2.5),
+    ),
+    _TypeDimensions(
+        type="bicycle",
+        height=_DimensionRange(0.3, 1.6),
+        width=_DimensionRange(0.1, 2.5),
+    ),
+    _TypeDimensions(
+        type="group_of_bicycles",
+        height=_DimensionRange(0.3, 1.6),
+    ),
+    _TypeDimensions(
+        type="wheelchair",
+        height=_DimensionRange(0.6, 1.6),
+        width=_DimensionRange(0.5, 1.6),
+    ),
+    _TypeDimensions(
+        type=["train", "rail_vehicle", "wagons"],
+        height=_DimensionRange(0.9, 6.0),
+        width=_DimensionRange(2.0, 900.0),
+    ),
```

Comment:
As the standard track gauge used in Europe is around 1.4m, there could conceivably be trains less wide than 2m (albeit unlikely). I suggest reducing the minimum as the goal is to detect outliers
```suggestion
    _TypeDimensions(
        type=["train", "rail_vehicle", "wagons"],
        height=_DimensionRange(0.9, 6.0),
        width=_DimensionRange(1.7, 900.0),
    ),
```
(1.7m worked well as a limit for testing in OSDaR23)

Reply:
Fixed in d827e1547ea7bb92e2ca616fbe728948bc9feeda